core/src/main/scala/com/evolutiongaming/kafka/flow/KafkaKey.scala

```diff
@@ -1,10 +1,15 @@
 package com.evolutiongaming.kafka.flow
 
+import cats.kernel.Hash
 import com.evolutiongaming.skafka.TopicPartition
 
 final case class KafkaKey(
   applicationId: String,
   groupId: String,
   topicPartition: TopicPartition,
   key: String
-)
\ No newline at end of file
+)
+
+object KafkaKey {
+  implicit val hash: Hash[KafkaKey] = Hash.fromUniversalHashCode
```

Comment:
I propose to add `KafkaKey` suffix: `hashKafkaKey`, as sometimes you may have weird compile problems because of conflicting types with same name

Reply:
Makes sense, done